### PR TITLE
OHRM5X-983: Time - Project & project activity details report screens

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        $fromDate = $request->query->get("fromDate");
+        $toDate = $request->query->get("toDate");
+        $includeTimesheet = $request->query->get("includeTimesheet");
+
+        if (!is_null($fromDate) && !is_null($toDate)) {
+            $component->addProp(new Prop('from-date', Prop::TYPE_STRING, $fromDate));
+            $component->addProp(new Prop('to-date', Prop::TYPE_STRING, $toDate));
+        }
+
+        if (!is_null($projectId)) {
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php:47]:
```suggestion
        if ($request->query->has('projectId')) {
```

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        $projectId = $request->query->getInt("projectId");
+        $activityId = $request->query->getInt("activityId");
+        $fromDate = $request->query->get("fromDate");
+        $toDate = $request->query->get("toDate");
+        $includeTimesheet = $request->query->get("includeTimesheet");
+
+        if (!is_null($fromDate) && !is_null($toDate)) {
+            $component->addProp(new Prop('from-date', Prop::TYPE_STRING, $fromDate));
+            $component->addProp(new Prop('to-date', Prop::TYPE_STRING, $toDate));
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php:44]:
```suggestion
        if ($request->query->has('fromDate') && $request->query->has('toDate')) {
            $component->addProp(new Prop('from-date', Prop::TYPE_STRING, $request->query->get('fromDate')));
            $component->addProp(new Prop('to-date', Prop::TYPE_STRING, $request->query->get('toDate')));
```

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                    Prop::TYPE_OBJECT,
+                    [
+                        "id" => 1,
+                        "label" => "Project Manhattan"
+                    ]
+                )
+            );
+        }
+
+        if (!is_null($activityId)) {
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php:61]:
```suggestion
        if ($request->query->has("activityId") {
```

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                    Prop::TYPE_OBJECT,
+                    [
+                        "id" => 1,
+                        "label" => "Develop timesheet component"
+                    ]
+                )
+            );
+        }
+
+        if (!is_null($includeTimesheet) && $includeTimesheet=="onlyApproved") {
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php:75]:
```suggestion
        if ($request->query->has('includeTimesheet') && $request->query->get('includeTimesheet') == 'onlyApproved') {
```

**Review by RajithaKumara — APPROVED** [review]

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php (hunk 1 above)
Comment (Super-Chama) [reply symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php]: done

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php (hunk 2 above)
Comment (Super-Chama) [reply symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php]: done

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php (hunk 3 above)
Comment (Super-Chama) [reply symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php]: done

File: symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php (hunk 4 above)
Comment (Super-Chama) [reply symfony/plugins/orangehrmTimePlugin/Controller/ActivityDetailsReportController.php]: done

**Pull request merged**